Playground.playground/Contents.swift

```diff
@@ -2,4 +2,15 @@
 
 import UInt128
 
-let integer: UInt128 = 0xf
\ No newline at end of file
+
+
+
+var integer: UInt128 = 0xf
+
+print(integer)
```

Comment:
👎 

I don't see how these changes to the Playground make it any more useful. Now, if the playground was updated to act as a nice walkthrough to show how to use this library, that would be a whole other story.